### ISSUE TITLE
I am not given a lance with Mastery of Melee (Kirin Tor)

### ISSUE THREAD

**Issue by UncleCrust** [dalaranwow/dalaran-wow#5868]
Original ticket: I accept Mastery of Melee from Magister Edian Sunhollow in which he says, "Take this lance...", but there is no lance in my inventory after accepting. I've restarted the game to no avail.

Golganeth: The NPC isn't supposed to provide the lance upon picking up the quest, you must take it.
Closing since the quest works as intended.

But exactly HOW does one "take it"? I've abandoned & re-accepted the quest 1/2 a dozen times & I cannot figure out how to take the lance. There is no lootable lance in the entire tent & none of the npcs has one. Please explain/detail. Thanks.

**Comment by alexuktc** [comment]
There are lance racks all over the Argent Tournament.

**Comment by UncleCrust** [comment]
Nope. I can't lot any of the Lance tracks around town. Icecrown sucks
majorly for quest completion... MAJORLY!

**Comment by Chenocide** [comment]
Maybe if you stopped playing on the dead and outdated realm, you wouldn't have these problems.

**Comment by UncleCrust** [comment]
Ya. Thanks, mate. I'm 68 & 3 corners and I just wanna play. I hate getting
ganked by opposing faction ... I just wanna play. Even here I ran into an
Alliance crew that had taken over Sholazar. I to stop playing for a while
cuz, no matter where I went, they would gank me. Sigh.

**Comment by Chenocide** [comment]
It's obviously your choice but I personally don't think it's worth it to throw out all social interactions and literally 4+ years of bug fixes, just so you don't get ganked while leveling, not to mention a population difference close 1:100.

**Comment by UncleCrust** [comment]
Lol. You're making it more & more attractive! ☺️ Thanks.
